.buildkite/scripts/steps/functional/security_serverless_osquery.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+source .buildkite/scripts/common/util.sh
```

Comment:
I think we can replace this with
```suggestion
source .buildkite/scripts/steps/functional/common.sh
```
and remove lines 8 and 9.  Sorry about the earlier suggestion, I got the abstraction layers mixed up.